### PR TITLE
Add more tests to timer-util

### PR DESCRIPTION
**Pull request by Prabhakar-Poudel**
Added remaning tests in timer utility function
Updated function to handle edge cases
Also fixed a couple minor issues

### CODE REVIEW

File: client/src/components/timer/Timer.jsx [hunk 1]
```diff
@@ -5,7 +5,8 @@ import './timer.css';
 
 class Timer extends PureComponent {
   render() {
-    const { time, className } = this.props;
+    const { time, classes } = this.props;
```
Comment (nimish-gupta) [inline client/src/components/timer/Timer.jsx]: This can be a functional component

**Review by nimish-gupta — CHANGES_REQUESTED** [review]
LGTM after changes

**Comment by Prabhakar-Poudel** [comment]
Closes #17
